### PR TITLE
RT-620-Banner-title

### PR DESCRIPTION
**Pull request by yashpmohod**
## Overview
  - Updated a env.js for baner-title now we are able to get baner-title from environment    

image

## How it was tested
Tested Using run cardio-patient app locally.

## Checklist

- [x] The title contains a short meaningful summary
- [x] I have added a link to this PR in the Jira issue
- [x] I have described how this was tested
- [ ] I have included screen shots for changes that affect the user interface
- [ ] I have updated unit tests
- [x] I have run unit tests locally
- [ ] I have updated documentation (including README)

### CODE REVIEW

File: projects/cardiovascular-patient-app/src/assets/env.js [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+//   clientId: "2dd5083f-85ba-4966-8b04-3c99dcd26601",
+//   appTitle: "CDSIC Blood Pressure App"
+// };
+
+
+(function (window) {
+  window["env"] = window["env"] || {};
+  window["env"]['clientId'] = "2dd5083f-85ba-4966-8b04-3c99dcd26601";
+  window["env"]['appTitle'] = "CDSIC Blood Pressure";
+ })(this);
```
Comment (semicolin) [inline projects/cardiovascular-patient-app/src/assets/env.js:15]: I don't think all of this extra cruft is needed. It works fine for me with
```js
window.env = {
  clientId: "2dd5083f-85ba-4966-8b04-3c99dcd26601",
  appTitle: "CDSiC Blood Pressure",
};
```

File: projects/cardiovascular-patient-app/src/environments/environment.logica-open.ts [hunk 2]
```diff
@@ -3,6 +3,10 @@ import { fhirclient } from 'fhirclient/lib/types';
 /**
  * A local test environment that uses Logica Sandbox open FHIR server with a hard-coded patient ID
  */
+declare global {
+  interface Window { env: any; }
+}
+
```
Comment (semicolin) [inline projects/cardiovascular-patient-app/src/environments/environment.logica-open.ts:9]: You can put this in `assets/env.d.ts`. You don't need to repeat it everywhere. You can also add stronger type safety:
```ts
declare global {
  interface Window {
    env: {
      clientId: string;
      appTitle: string;
    };
  }
}
export {};
```

**Review by semicolin — CHANGES_REQUESTED** [review]

**Review by semicolin — CHANGES_REQUESTED** [review]
I think you can fix the unit tests by loading env.js in karma.conf:
```js
module.exports = function (config) {
  config.set({
    ...
    // load run-time environment vars
    files: ['src/assets/env.js'],
```

File: projects/cardiovascular-patient-app/src/assets/env.template.js [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
+// window.env = {
+//     clientId: "${CLIENT_ID}",
+//     appTitle: "${APP_TITLE}"
+//   };
+
+(function (window) {
+  window["env"] = window["env"] || {};
+  window["env"]['clientId'] = "${CLIENT_ID}";
+  window["env"]['appTitle'] = "${APP_TITLE}";
+})(this);
```
Comment (semicolin) [inline projects/cardiovascular-patient-app/src/assets/env.template.js:10]: please clean up env.template.js so it matches the local env.js

**Review by semicolin — CHANGES_REQUESTED** [review]

File: projects/cardiovascular-patient-app/src/assets/env.template.js (hunk 3 above)
Comment (yashpmohod) [reply projects/cardiovascular-patient-app/src/assets/env.template.js:10]: Okay @semicolin

**Review by semicolin — APPROVED** [review]
Code looks good. Did you try running it in docker?

**Pull request merged**